conf/script/build-ha-io

```diff
@@ -620,10 +620,11 @@ hax_rsc_add() {
 motr_systemd_update() {
     log "${FUNCNAME[0]}: Adding Motr to Pacemaker..."
 
-    cmd='
-    sudo sed "s/TimeoutStopSec=.*/TimeoutStopSec=5sec/"
-        -i /usr/lib/systemd/system/m0d@.service &&
-    sudo systemctl daemon-reload'
+    cmd="
+    sudo sed -e "s/TimeoutStopSec=.*/TimeoutStopSec=5sec/"
+             -e \"/ExecStart=/aExecStartPost=/bin/sh -c 'while [[ \$\(/opt/seagate/cortx/hare/libexec/get-process-state %i\) != M0_CONF_HA_PROCESS_STARTED ]]\; do sleep 1\; done'\"
```

Comment:
There is more proper way of doing this - by calling sd_notify(). But it was not done on purpose. From motr/m0d.c code:

```c
        /*
         * From the systemd's point of view, service can be considered as
         * started when it can handle incoming connections, which is already
         * true before m0_cs_start() is called. otherwise, if sd_notify() is
         * called after m0_cs_start() it leads to a deadlock, because different
         * m0d instances will wait for each other forever.
         */
        rc = sd_notify(0, "READY=1");
```

Reply:
@andriytk ,
Right, but if sd_notify with `READY=1` moved after `m0_cs_start()`, it will block stating all other Motr process till current service started.

Currently Hare start Motr process on a node in sequentially. 
m0_cs_start() will try to connect all other service endpoints which are not started yet.

Reply:
Right. So better to do this change in Motr rather than here. (If we are about to do it at all.) I think.